core/tests/protractor/collections.js

```diff
@@ -83,6 +81,32 @@ describe('Collections', function() {
     users.logout();
   });
 
+  it('interacts with the collection player as a guest', function() {
+    collectionPlayer.goToDemoCollection();
+    // Check that we are in the guest state (sign in button visible).
+    collectionPlayer.verifySignInButton();
```

Comment:
This doesn't seem particularly collectionPlayer-specific. Perhaps it should go in `general`? And maybe we can call it verifyUserIsLoggedOut().